Create a consolidated docs site that incorporates the v1 docs into the v2 site (#17)
FriendlyCaptcha/friendly-docs: docs/introduction/index.md

---
slug: /
---

# Introduction

Friendly Captcha is a service that protects websites from bots and abuse in a privacy-friendly and accessible way. To learn more about the company or service, you can visit the [Friendly Captcha website](https://friendlycaptcha.com).

This site contains the technical documentation that will help you integrate the Friendly Captcha's anti-abuse service into your website. Integration entails adding a widget to the websites you want to protect. The Friendly Captcha widget looks like this:

<figure style={{ textAlign: 'center' }}>
    <img src="/img/widget-v2-ready.png" alt="Screenshot of the Friendly Captcha widget" />
    <figcaption><i>Screenshot of the Friendly Captcha Widget</i></figcaption>
</figure>

## What's different about Friendly Captcha?

Other captcha systems like [Google reCAPTCHA](https://en.wikipedia.org/wiki/ReCAPTCHA) or [hCaptcha](https://hcaptcha.com) invent tasks for website users that are meant to be easy for humans but impossible for bots. An example is clicking on images that contain cars.

> These tasks are annoying for many users, but they can even create barriers to legitimate access. Users with difficulty seeing or hearing, or with advanced age, or who just lack technical savvy, might struggle to complete them.

Reviewer: ```suggestion
These tasks are annoying for many users, but they can be a barriers for legitimate users. Users with difficulty seeing or hearing, with advanced age, or who just lack technical savvy, might struggle to complete them.
```

(If you agree, feel free to ignore if you like the old version better)

Author: My personal opinion is that using the word "users" twice in that way feels a bit off. And, to be annoyingly pedantic, the phrasing "annoying to many users, but barrier to legitimate users" could be read to claim that users to whom the tasks are annoying but not a barrier are illegitimate. What about

>_These tasks are annoying for many users, and can even be a **barrier** to some of them_.
